packages/components/dropdowns/dropdown-menu/src/dropdown-menu.story.js

```diff
@@ -24,30 +24,31 @@ storiesOf('Components|Dropdowns|DropdownMenu', module)
       sidebar: Readme,
     },
   })
-  .add('DropdownMenu - List menu content', () => (
-    <Section align="center">
-      <DropdownMenu
-        triggerElement={<IconButton icon={<ColumnsIcon />} label="list" />}
-        menuHorizontalConstraint={select(
-          'menu horizontalConstraint',
-          Constraints.getAcceptedMaxPropValues(),
-          6
-        )}
-        menuPosition={select('Menu position', ['left', 'right'], 'left')}
-        menuType="list"
-      >
-        <DropdownMenu.ListMenuItem onClick={action('onClick')}>
-          Option 1
-        </DropdownMenu.ListMenuItem>
-        <DropdownMenu.ListMenuItem onClick={action('onClick')} isDisabled>
-          Option 2
-        </DropdownMenu.ListMenuItem>
-        <DropdownMenu.ListMenuItem onClick={action('onClick')}>
-          Option 3
-        </DropdownMenu.ListMenuItem>
-      </DropdownMenu>
-    </Section>
-  ))
+  .add('DropdownMenu - List menu content', () => {
+    const optionsCount = number('Options count', 5);
```

Comment:
This is to allow having long lists in the dropdown menu so the height and scrollability can be tested.

Reply:
💯